### PR TITLE
Helpful/Report components (refactor in future) 

### PR DESCRIPTION
**Pull request by murkymode**
Changed a lot of small things here, will be more encapsulated going forward.

### CODE REVIEW

File: client/src/components/Q&A/AddQuestion.jsx [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 import React from 'react';
 
 export default function AddQuestion({ setShow }) {
   return (
     <div
       className="add-question"
       style={{ border: '2px solid slateblue' }}
-    >
-      Hello from Add Question
+    >{}
```
Comment (jordanabbasi) [inline client/src/components/Q&A/AddQuestion.jsx]: typo?

File: client/src/components/Q&A/AnswerRow.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
-/* eslint-disable import/extensions */
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import Report from './Report';
 
-
-export default function AnswerRow({ questionId }) {
+export default function AnswerRow({ questionId, incrementCount }) {
```
Comment (jordanabbasi) [inline client/src/components/Q&A/AnswerRow.jsx]: take out incrementCount?

Comment (jordanabbasi) [reply client/src/components/Q&A/AnswerRow.jsx]: You need a separate counter for # of answers.

File: client/src/components/Q&A/SearchBar.jsx [hunk 3]
```diff
@@ -1,12 +1,11 @@
-/* eslint-disable import/extensions */
 import React from 'react';
 
 export default function SearchBar({ setSearchTerm }) {
   function handleSearch(event) {
     const search = event.target.value;
     if (search.length >= 3) {
       setSearchTerm(search);
-    } else if (search.length === 0) {
+    } else if (search.length < 3) {
```
Comment (jordanabbasi) [inline client/src/components/Q&A/SearchBar.jsx]: just an else?

**Review by jordanabbasi — CHANGES_REQUESTED** [review]
Minor changes suggested.

File: client/src/components/Q&A/AddQuestion.jsx (hunk 1 above)
Comment (murkymode) [reply client/src/components/Q&A/AddQuestion.jsx]: Indeed! Removed

File: client/src/components/Q&A/AnswerRow.jsx (hunk 2 above)
Comment (murkymode) [reply client/src/components/Q&A/AnswerRow.jsx]: Nice catch! Removed as well

File: client/src/components/Q&A/SearchBar.jsx (hunk 3 above)
Comment (murkymode) [reply client/src/components/Q&A/SearchBar.jsx]: Ah, yes

**Review by jordanabbasi — APPROVED** [review]
Looks good!

**Pull request merged**